src/aria2.cpp

```diff
@@ -55,12 +55,12 @@ void pauseAnyActiveDownloads(const std::string& ariaSessionFilePath)
 
 } // unnamed namespace
 
-Aria2::Aria2():
+Aria2::Aria2(std::string downloadDir):
   mp_aria(nullptr),
   m_port(42042),
-  m_secret(getNewRpcSecret())
+  m_secret(getNewRpcSecret()),
+  m_downloadDir(downloadDir)
```

Comment:
`m_downloadDir` can be dropped - it is private and is used only in the constructor